packages/macro/tests/index.test.ts

```diff
@@ -66,5 +76,14 @@ pluginTester({
         const _dynamicImport_ = null;
       }
     `,
+    "client-side with skip option": `
+      import { useSSRComputation } from "../lib/index.macro"
+      const skip = true;
+      const x = useSSRComputation("./a.ssr-computation", [], { skip, webpackChunkName: "custom-chunk-name" })
+    `,
+    "client-side with serverSideOnly option": `
+      import { useSSRComputation } from "../lib/index.macro"
+      const x = useSSRComputation("./a.ssr-computation", [], { webpackChunkName: "custom-chunk-name", serverSideOnly: true })
+    `,
```

Comment:
Can we split snapshot tests into multiple files (one file for every distinct feature containing client and server tests)?

Reply:
The idea is to make small `ts.snap` files, each showing off one distinct feature: how the feature works server-side, and what it does client-side